lib/src/kvs/tx.rs

```diff
@@ -1557,6 +1558,29 @@ impl Transaction {
 		})
 	}
 
+	/// Retrieve all model definitions for a specific database.
+	pub async fn all_db_models(
+		&mut self,
+		ns: &str,
+		db: &str,
+	) -> Result<Arc<[DefineModelStatement]>, Error> {
+		let key = crate::key::database::ml::prefix(ns, db);
+		Ok(if let Some(e) = self.cache.get(&key) {
+			if let Entry::Mls(v) = e {
+				v
+			} else {
+				unreachable!();
```

Comment:
If this really is unreachable, why not skip the `else` branch?

Reply:
I knew that but we can merge those two if lets which would get rid of the unreachable

```rust
if let Some(Entry::Mls(v)) = self.cache.get(&key) {
    v
}
```

Reply:
Ah yes, good point. Let's do that in a separate PR!

Reply:
There are many functions with similar code...